src/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import Card from "@mui/material/Card";
+import { AppBar, Button, Divider, IconButton, Toolbar } from "@mui/material";
+import Box from "@mui/material/Box";
+
+export const Page1Card = styled(Card)({
+  maxWidth: "250px",
+  marginTop: "100px",
+  marginRight: "40px",
+  border: "1px solid #CAC4D0",
+  borderRadius: "12px",
+});
+
+export const Page1Button = styled(Button)({
+  borderRadius: "18px",
+  marginLeft: "auto",
+  height: "30px",
+  marginTop: "20px",
+  fontSize: "10px",
+  "&.contained": {
+    backgroundColor: "#6750A4",
+  },
+});
```

Comment:
All the buttons are the same throughout the entire application. All you needed to do was to create a new theme and specify the palette colors. This was quite unnecessary.